doc/source/GettingStarted/TutorialAssemblingTheGafferBot/index.md

```diff
@@ -176,62 +176,62 @@ When a scene node computes a scene, scene data passes through it like so:
 
 ### Plugs ###
 
-The inputs and outputs of a node are called **plugs**, and are represented in the _Graph Editor_ as colored circles around the node's edges.
+The inputs and outputs of a node are called **plugs**, and are represented in the Graph Editor as colored circles around the node's edges.
 
 <!-- TODO: add close-up of node plugs ![](images/nodePlug.png "A node's plug") -->
 
-For your two nodes to occupy the same scene (and later render together), you will need to combine them into a single scene. You can connect both of their output plugs to a Group node, and you can also rearrange the nodes to better visually represent the data flow in the graph.
+For your two nodes to occupy the same scene (and later render together), you will need to combine them into a single scene. You can connect both of their Out plugs to a Group node, and you can also rearrange the nodes to better visually represent the data flow in the graph.
 
 
 ## Connecting plugs ##
 
 It's time to connect the SceneReader and Camera nodes to combine their scenes:
 
-1. Click the background of the _Graph Editor_ to deselect all nodes.
+1. Click the background of the Graph Editor to deselect all nodes.
 
 2. Create a Group node (_Scene_ > _Hierarchy_ > _Group_).
 
-3. Click and drag the SceneReader node's _out_ plug (at the bottom; blue) onto the Group node's _in0_ plug (at the top; also blue). As you drag, a node connector will appear. Once you connect them, a second input plug (_in1_) will appear on the Group node, next to the first.
+3. Click and drag the SceneReader node's Out plug (at the bottom; blue) onto the Group node's In0 plug (at the top; also blue). As you drag, a node connector will appear. Once you connect them, a second input plug (In1) will appear on the Group node, next to the first.
```

Comment:
`Out` and `In#` plugs aren't capitalised in the UI:

![image](https://user-images.githubusercontent.com/896779/72746377-e07cab80-3ba9-11ea-9b64-879a61ae2b8b.png)
![image](https://user-images.githubusercontent.com/896779/72746380-e5d9f600-3ba9-11ea-9280-7d501de04a28.png)